ui/src/views/compute/EditVM.vue

```diff
@@ -328,10 +328,8 @@ export default {
         params.name = values.name
         params.displayname = values.displayname
         params.ostypeid = values.ostypeid
-        if (this.securityGroupsEnabled) {
-          if (values.securitygroupids) {
-            params.securitygroupids = values.securitygroupids
-          }
+        if (this.securityGroupsEnabled && Array.isArray(values.securitygroupids) && values.securitygroupids.length > 0) {
+          params.securitygroupids = values.securitygroupids
```

Comment:
does this work? I saw a similar code block yesterday where we were using it like the following,
`params.securitygroupids = values.securitygroupids.join(',')`

Reply:
I tested this in QA, @shwstppr. The only thing not working is clearing the list of security groups completely. Which, I think, is a separate issue.
What do you think might happen when not calling `join(‘,’)`?

Reply:
@DaanHoogland, that was just a question from my side. I've not tested it and do not know if it makes any difference at the moment. If it is already tested then :+1: